app/vmalert/remotewrite/remotewrite.go

```diff
@@ -183,6 +183,7 @@ func (c *Client) run(ctx context.Context) {
 var (
 	sentRows            = metrics.NewCounter(`vmalert_remotewrite_sent_rows_total`)
 	sentBytes           = metrics.NewCounter(`vmalert_remotewrite_sent_bytes_total`)
+	sentDuration        = metrics.NewFloatCounter(`vmalert_remotewrite_sent_duration_seconds_total`)
```

Comment:
```suggestion
	sendDuration        = metrics.NewFloatCounter(`vmalert_remotewrite_send_duration_seconds_total`)
```

it should be `send`, since it is a process)